pkg/router/template/router.go

```diff
@@ -131,6 +132,8 @@ type templateData struct {
 	State map[string](ServiceAliasConfig)
 	// the service lookup
 	ServiceUnits map[string]ServiceUnit
+	// number of service endpoints
+	ServiceEndpoints map[string]int
```

Comment:
It is a convenience map it seems. Can we just not have it and use len(svc.EndpointTable) instead? Or am I missing something?